stdlib/ctypes/wintypes.pyi

```diff
@@ -186,55 +186,53 @@ class WIN32_FIND_DATAW(Structure):
     cFileName: _CField[Array[WCHAR], str, str]
     cAlternateFileName: _CField[Array[WCHAR], str, str]
 
-# These pointer type definitions use _Pointer[...] instead of POINTER(...), to allow them
-# to be used in type annotations.
```

Comment:
I removed the comment because it seemed redundant: POINTER() is used to define these at runtime, but it's a runtime function. That's not appropriate for use in typeshed by definition.